Dashboard/app/js/templates/navData/monitoring-data.handlebars

```diff
@@ -157,9 +157,15 @@
                             {{else}}
                                 {{t _no_signature_found}}
                             {{/if}}
-                        {{else}}
+                        {{else}} {{#if view.isCaddisflyType}}
```

Comment:
I think it is not that bad actually - all the 'else' blocks are aligned. This should really be a 'case' block, but that is not available in handlebar files. The alternative is an 8-fold indented block, which I am not sure would improve it.